Tests/Tests/STPPaymentIntentTest.m

```diff
@@ -171,18 +178,35 @@ - (void)testDecodedObjectFromAPIResponseMapping {
     XCTAssertEqualObjects(paymentIntent.stripeDescription, @"My Sample PaymentIntent");
     XCTAssertFalse(paymentIntent.livemode);
     XCTAssertEqualObjects(paymentIntent.receiptEmail, @"danj@example.com");
+    
+    // Deprecated: nextSourceAction
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
     XCTAssertNotNil(paymentIntent.nextSourceAction);
```

Comment:
It *might* be interesting to also assert object identity equality, since we know that `nextSourceAction` should just call `nextAction` (and similar for the `authorizeWithUrl`/`redirectToURL` pair):

    XCTAssertEqual(paymentIntent.nextSourceAction, paymentIntent.nextAction, @"Should be same object");

Doing this kinda makes the parts of `STPRedirectContextTest.m` that test the deprecated properties *and* new properties superfluous.

Reply:
Nice, I like it